lib/build-metadata.js

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env node
+"use strict";
+
+/**
+ * @typedef {Object} metadata
+ * @property {metadataProperty} *
```

Comment:
cwsShelfUpload isn't a namespace. Its also in reference to the code base I am in right now. It would only make sense if I added namespaces for everything.

Should I add a namespace for this? Probably should.